src/components/molecules/Alert/Alert.tsx

```diff
@@ -56,31 +40,63 @@ export const Alert = ({
         }
     };
 
-    return isVisible ? (
+    const alertIcon = alertIconMapping[severity];
+
+    if (!isVisible) return null;
+
+    return (
         <section
             aria-label={severity}
             role='alert'
-            className={clsx('rounded-xl', {
-                'border-2 border-yellow bg-yellow/20': variant === 'outlined', // should be changed to handle different colors
-                'bg-[rgba(41, 45, 63, 0.60)] border border-white-10 shadow-tab':
-                    variant === 'solid',
-            })}
+            className={clsx(
+                'flex w-full flex-row items-start justify-between gap-1 rounded-md border px-2.5 text-2xs text-neutral-50 laptop:text-xs',
```

Comment:
right padding is 8px

Reply:
designs do not suggest a font size change between mobile and desktop

Reply:
think these fonts sizes and line heights should move to subtitle for clarity

Reply:
the gap is 6px

Reply:
I made the assumption that the 8px right padding was an oversight on design, but it makes sense now. Will change it to 8px

Reply:
<img width="362" alt="Screenshot 2024-05-09 at 10 09 41 AM" src="https://github.com/Secured-Finance/secured-finance-app/assets/92713298/4ab508eb-3282-4140-be40-d5537b79e103">

I see that the gap is also 8px on desktop. I'll change it to gap-2.